tests/index.js

```diff
@@ -47,21 +46,17 @@ describe('broccoli-jscs', function() {
       var sourcePath = 'tests/fixtures/issue-found';
       chdir(sourcePath);
 
-      var tree = new jscsTree('.', {
-        logError: function() {}
-      });
+      var tree = new jscsTree('.');
 
-      builder = new broccoli.Builder(tree);
-      return builder.build().then(function() {
-        expect(tree.bypass).to.not.be.ok();
-      });
+      expect(tree.bypass).to.not.be.ok();
     });
 
     it('uses the jscsrc as configuration', function() {
       var sourcePath = 'tests/fixtures/issue-found';
-      chdir(sourcePath);
 
-      var tree = new jscsTree('.', {
+      var tree = new jscsTree(sourcePath, {
+        persist: false,
+        configPath: path.join(sourcePath, '.jscsrc'),
```

Comment:
I believe this changes the meaning of the test, where it is saying you don't need to supply the name of the file, we will auto detect.